Packs/MicrosoftGraphSecurity/Integrations/MicrosoftGraphSecurity/MicrosoftGraphSecurity_description.md

```diff
@@ -20,10 +12,14 @@ There are two application authentication methods available:
 
 Depending on the authentication method that you use, the integration parameters might change.
 
-To allow us to access to Microsoft Graph Security, an admin has to approve our app using an admin consent flow, by clicking this [link](https://oproxy.demisto.ninja/ms-graph-security).
+To use the **Cortex XSOAR application** and allow Cortex XSOAR access to Microsoft Graph Security an administrator has to approve our app using an admin consent flow by clicking this **[link](https://oproxy.demisto.ninja/ms-graph-security)**.
 After authorizing the Cortex XSOAR app, you will get an ID, Token, and Key which should be inserted in the integration instance settings fields.
 If you previously had an API V1 configured based on the credentials obtained from this method, refer to the link above to gain new credentials with the relevant permissions.
 
+### Important
+1. The *eDiscovery* and *Threat Assessment* commands are only supported when using the `Authorization Code flow` with `Delegated (work or school account)` permission type.
+2. When using `Authorization Code flow`, the connection should be tested using the `!msg-auth-test` command.
```

Comment:
Recommend:
 When using `Authorization Code flow`, the connection should be tested using the ***!msg-auth-test*** command.